test/opml.cpp

```diff
@@ -302,7 +302,7 @@ TEST_CASE("import() tags from category attribute", "[Opml]")
 {
 	FileUrlReader urlcfg;
 	REQUIRE_NOTHROW(
-		opml::import("file://" + utils::getcwd() + "/data/category.opml", urlcfg)
+		opml::import("file://" + std::string(utils::getcwd()) + "/data/category.opml", urlcfg)
```

Comment:
`std::string()` is a temporary kludge; don't use it explicitly. Use `.to_locale_string()` instead.

Ditto in all other places below except the ones where I mention `.display()`.